deploy.sh

```diff
@@ -383,7 +383,36 @@ function question {
 
   if [[ $options == "-sensitive" ]]; then
     log "> $message"
-    read -s value
+    # read -s value
+    # echo off
+    stty -echo    
+    #read password
+    charcount=0
+    while IFS= read -p "$prompt" -r -s -n 1 ch
+    do
+        # Enter - accept password/pull-secret
+        if [[ $ch == $'\0' ]] ; then
+            break
+        fi
+        # Backspace
+        if [[ $ch == $'\177' ]] ; then
+            if [ $charcount -gt 0 ] ; then
+                charcount=$((charcount-1))
+                prompt=$'\b \b'
+                value="${value%?}"
+            else
+                PROMPT=''
```

Comment:
what is `PROMPT`?

Reply:
prompt is used instead of read! prompts users for password/pull-secret. Script also handles backspace. Let me know if any changes needed. Also checking if I can make it simple.